### ISSUE TITLE
Expose CreateChannel command via intent

### ISSUE THREAD

**Issue by tanya-coding** [atomist/lifecycle-automation#81]
It might be useful to be able to run a command that creates a channel and links it to an existing repo. We have the command but we don't specify intent: https://github.com/atomist/lifecycle-automation/blob/3b304f32cd907ce3c6b2b758a191151fa061b125/src/handlers/command/slack/CreateChannel.ts

This came up in support channel in Atomist community: https://atomist-community.slack.com/archives/C29GYTYDC/p1515003793000391

**Comment by cdupuis** [comment]
Added to master with #82

**Comment by cbeams** [comment]
It's impressive on a couple levels that I suggested this feature only hours ago in the Slack chat linked above, and that the functionality is now already available. Nice, and thank you!

This is a screenshot of my first attempt to use `@atomist link channel`. Note how it fails:

![image](https://user-images.githubusercontent.com/301810/34542711-fc87d246-f0dd-11e7-8b8b-b0884e5adb74.png)

Here's the text of that error message:

```
Unable to run command
Unsuccessfully invoked command-handler CreateChannel of @atomist/lifecycle-automation@0.2.14-20180103211015
Correlation ID: 858f5e23-4633-43ef-ae5d-29dd1fcee76d
```

Note that the #libdohj channel was created, but that the linkage between it and the bisq-network/libdohj repo had not been established (at least I think it hadn't, because I invited `@atomist` and it asked me if I wanted to link the repo, as if it had not already been done).
